docs/deployment/file-access-auth.md

```diff
@@ -0,0 +1,159 @@
+---
+title: File Access Authorization
+parent: Deployment
+nav_order: 4
+---
+
+# (BETA) File Access Authorization
+
+> **IMPORTANT:** This feature is currently in beta. Configuration and log formats are subject to change.
+
+> **IMPORTANT:** This feature is only supported on macOS 13 and above.
+
+File Access Authorization allows admins to configure Santa to monitor filesystem paths for potentially unwanted access and optionally deny the operation.
+
+## Enabling the Feature
+
+To enable this feature, the `FileAccessPolicyPlist` key in the main Santa configuration must contain the path to a configuration file (format specified below).
+
+## Configuration
+
+| Key                 | Parent       | Type       | Required | Default    | Description |
+| ------------------- | ------------ | ---------- | -------- | ---------- | ----------- |
+| `Version`           | `<Root>`     | String     | Yes      | N/A        | Version of the configuration. Will be reported in events. |
+| `WatchItems`        | `<Root>`     | Dictionary | No       | N/A        | The set of configuration items that will be monitored by Santa. |
+| `<Name>`            | `WatchItems` | Dictionary | No       | N/A        | A unique name that identifies a single watch item rule. This value will be reported in events. The name must be a legal C identifier (e.g. must conform to the regex `[A-Za-z_][A-Za-z0-9_]*`). |
```

Comment:
```suggestion
| `<Name>`            | `WatchItems` | Dictionary | No       | N/A        | A unique name that identifies a single watch item rule. This value will be reported in events. The name must be a legal C identifier (e.g., must conform to the regex `[A-Za-z_][A-Za-z0-9_]*`). |
```

Reply:
There's a mix of e.g. or i.e. with following commas. I'm adding the comma for consistency as that's what I see in other docs (feel free to go either way though).